### PR TITLE
Fix/a few minor fixes

### PR DESCRIPTION
**Pull request by RobjS**

### CODE REVIEW

File: bin/wpc_init [hunk 1]
```diff
@@ -114,7 +114,7 @@ CONTAINER=`docker-compose ps -q wordpress`
 #
 # Issue: https://github.com/docker/compose/issues/3352
 
-docker exec -i ${FLAGS} ${CONTAINER} wp ${@}
+docker exec -i ${FLAGS} ${CONTAINER} wp "${@}"
```
Comment (mallorydxw) [inline bin/wpc_init:117]: Tested. This works.

File: README.md [hunk 2]
```diff
@@ -39,3 +39,11 @@ WordPress listens on port 80. Mailcatcher listens on port 1080. Beanstalkd Conso
 MySQL is available by running `./bin/wp db cli`. It will also be listening on port 3306.
 
 To access bash within the container: `./bin/console`.
+
+## Common issues
+
+All the ports that docker will be listening on (80, 1080, 2080, 3066 & 11300) need to be available, so if you've got local processes running on them, that will stop the containers starting up, usually with an `EADDRINUSE` error.
+
+Things that might be running on port 80: `apachectl` (stop with `sudo apachectl stop`), any other local server (e.g. `whippet-server`, XAMPP)
```
Comment (mallorydxw) [inline README.md]: I think the first `apachectl` should just be `apache`?

File: bin/wpc_init [hunk 3]
```diff
@@ -195,7 +195,14 @@ then
...
     done
   else
-    echo "\033[96mWarning:\033[0m Cannot import content as wordpress-importer plugin is not installed"
+    echo "WordPress Importer not installed... installing now"
+    wp plugin install wordpress-importer --activate
+    for file in $content/*.xml
+    do
+      echo "Importing $file..."
+      wp import $file --authors=skip
+    done
```
Comment (mallorydxw) [inline bin/wpc_init]: The above 5 lines are repeated. Can we DRY them?

**Review by mallorydxw — CHANGES_REQUESTED** [review]
Everything works. Just a couple of small things.

**Comment by RobjS** [comment]
Made those changes, ready for another look now.

**Review by mallorydxw — APPROVED** [review]
👍

**Pull request merged**